commit 47e3564bbdf1c244fe382a350e9852d6c11bbe40 (from 02550af7c3a5a17ee158a620c3625299c75876b6)
Merge: 02550af ca6c6ec
Author: Roxy-3DPrintBoard <Roxy-3DPrintBoard@users.noreply.github.com>
Date:   Mon Apr 11 15:58:05 2016 -0500

    Merge pull request #3462 from jbrazio/bugfix/old-ide-warning-followup
    
    Versions of Arduino IDE prior to 1.6.0 are no longer supported

diff --git a/Marlin/SanityCheck.h b/Marlin/SanityCheck.h
index b1a9c7b..901da3b 100644
--- a/Marlin/SanityCheck.h
+++ b/Marlin/SanityCheck.h
@@ -34,8 +34,8 @@
  * release we will stop supporting old IDE versions and will require user
  * action to proceed with compilation in such environments.
  */
-#if !defined(ARDUINO) || ARDUINO < 10500
-  #warning Versions of Arduino IDE prior to 1.5 are no longer supported, please update your toolkit.
+#if !defined(ARDUINO) || ARDUINO < 10600
+  #error Versions of Arduino IDE prior to 1.6.0 are no longer supported, please update your toolkit.
 #endif
 
 /**

